cmd/nerdctl/container/container_create.go

```diff
@@ -409,12 +409,17 @@ func processContainerCreateOptions(cmd *cobra.Command) (types.ContainerCreateOpt
 	if err != nil {
 		return opt, err
 	}
+	quiet, err := cmd.Flags().GetBool("quiet")
```

Comment:
Should be a separate PR

Reply:
Does Docker or Podman have this flag?

Otherwise —pull-quiet might be less confusing? idk

Reply:
👍🏼 Opened https://github.com/containerd/nerdctl/pull/3551
Docker has --quiet across load, run, and create.